arnold/crud/read/step.py

```diff
@@ -87,7 +90,7 @@ def find_sample_fields(adapter: ArnoldAdapter) -> list[str]:
 def find_step_type_udfs_pipe(
```

Comment:
Use get since it returns something.

Reply:
I will do the step refactoring in a separate PR - since it was a bit bigger than the others and more complex